heft-plugins/heft-typescript-plugin/src/TypeScriptBuilder.ts

```diff
@@ -1297,12 +1297,16 @@ export class TypeScriptBuilder {
   }
 }
 
+type BuilderProgramWithInternals = TTypescript.BuilderProgram & {
+  state?: { changedFilesSet: Set<string> };
+  getState(): { changedFilesSet: Set<string> };
```

Comment:
Include a doc comment noting that `state` was removed in 5.6 and that `getState` is the replacement.

Reply:
(the opposite but yeah)